src/commands/bootstrap/index.ts

```diff
@@ -0,0 +1,685 @@
+import { create } from '../projects.js';
+import { execSync } from 'child_process';
+import { trackEvent } from '../../analytics.js';
+
+export const command = 'create-app';
+export const aliases = ['bootstrap'];
+export const describe = 'Initialize a new Neon project';
+
+export const builder = (yargs: yargs.Argv) =>
+  yargs.option('context-file', {
+    hidden: true,
+  });
+
+const onPromptState = (state: {
+  value: InitialReturnValue;
+  aborted: boolean;
+  exited: boolean;
+}) => {
+  if (state.aborted) {
+    // If we don't re-enable the terminal cursor before exiting
+    // the program, the cursor will remain hidden
+    process.stdout.write('\x1B[?25h');
+    process.stdout.write('\n');
+    process.exit(1);
+  }
+};
+
+export const handler = async (args: CommonProps) => {
+  await bootstrap(args);
+};
+
+type BootstrapOptions = {
+  auth?: 'auth.js';
+  framework: 'Next.js' | 'SvelteKit' | 'Nuxt.js';
+  deployment: 'vercel' | 'cloudflare';
+  orm?: 'drizzle' | 'prisma';
+  packageManager: 'npm' | 'pnpm' | 'bun' | 'yarn';
+};
+
+// `getCreateNextAppCommand` returns the command for creating a Next app
+// with `create-next-app` for different package managers.
+function getCreateNextAppCommand(
+  packageManager: BootstrapOptions['packageManager'],
+) {
+  switch (packageManager) {
+    case 'npm':
+      return 'npx create-next-app';
+    case 'bun':
+      return 'bunx create-next-app';
+    case 'pnpm':
+      return 'pnpm create next-app';
+    case 'yarn':
+      return 'yarn create next-app';
+  }
+}
+
+function getExecutorProgram(
+  packageManager: BootstrapOptions['packageManager'],
+) {
+  switch (packageManager) {
+    case 'npm':
+      return 'npx';
+    case 'pnpm':
+      return 'pnpx';
+    case 'bun':
+      return 'bunx';
+    case 'yarn':
+      return 'yarn exec';
+  }
+}
+
+function getGlobalInstallProgram(
+  packageManager: BootstrapOptions['packageManager'],
+  packageName: string,
+) {
+  switch (packageManager) {
+    case 'npm':
+      return `npm install -g ${packageName}`;
+    case 'pnpm':
+      return `pnpm install -g ${packageName}`;
+    case 'bun':
+      return `bun add -g ${packageName}`;
+    case 'yarn':
+      return `yarn global add ${packageName}`;
+  }
+}
+
+const bootstrap = async (props: CommonProps) => {
+  const out = writer(props);
+
+  if (isCi()) {
+    throw new Error('Cannot run interactive auth in CI');
+  }
+
+  const res = await prompts({
+    onState: onPromptState,
+    type: 'text',
+    name: 'path',
+    message: 'What is your project named?',
+    initial: 'my-app',
+    validate: (name) => {
+      const validation = validateNpmName(basename(resolve(name)));
+      if (validation.valid) {
+        return true;
+      }
+      return 'Invalid project name: ' + validation.problems[0];
+    },
+  });
+  trackEvent('create-app', { phase: 'start' });
+
+  let projectPath;
+  if (typeof res.path === 'string') {
```

Comment:
Reversing the  "if statement" would get rid of "else" branch and "let" statement